pkg/fuzzing/fuzz.go

```diff
@@ -0,0 +1,35 @@
+package fuzzing
+
+import "encoding/json"
+
+type Product struct {
+	ID          string  `json:"id" bson:"_id"`
+	Name        string  `json:"name" validate:"required"`
+	Description string  `json:"description"`
+	Price       float32 `json:"price" validate:"gt=0"`
+	SKU         string  `json:"sku" validate:"required,sku"`
+	CreatedOn   string  `json:"-"`
+	UpdatedOn   string  `json:"-"`
+}
+
+func Fuzz(data []byte) int {
```

Comment:
Can you add some documentation in a readme or something in this directory on how to use the package

Like running the go-fuzz command or at least have a link to the package somewhere